### ISSUE TITLE
Surfel not removing moving objects

### ISSUE THREAD

**Issue by Myzhar** [victorprad/InfiniTAM#134]
Hello, 
I'm trying the Surfel engine and I noticed that moving objects are not removed like it happens in dense engine.

Is there any configuration to be set to correctly remove moving objects?

**Comment by sgolodetz** [comment]
See https://github.com/victorprad/InfiniTAM/issues/78.

The surfel engine is something of a beta unfortunately - it works, but there are certain things that aren't currently implemented.

**Comment by Myzhar** [comment]
Sorry for the duplicated question... I have not see the #78.
Thank you

**Comment by daalberti1** [comment]
Hi, thank you very much for the amazing code!

I'm sorry but I don't undertstand if InfiniTAM (v3) should be able to reconstruct scenes with moving objects. I thought no, but then I read @Myzhar's question talking about a _dense engine_ which is able to remove moving objects... I'm wondering what he's referring to.

I tried the 3 different engines changing `internalSettings->libMode = ITMLibSettings::LIBMODE_LOOPCLOSURE;` in infiniTAM.cpp at line 199, but none was able to work with the _umbrella dataset_ from https://cloud9.cs.fau.de/index.php/s/46qcNZSNePHx08A .

![Screenshot from 2022-01-21 10-27-48](https://user-images.githubusercontent.com/74023873/150502092-e400f5d1-6a65-4b3b-95d4-b4684890a2ff.png)

I noticed that with `LIBMODE_BASIC` InfiniTAM is able at least to handle the really first small movements of the object, while with `LIBMODE_LOOPCLOSURE` the output is completely black after just a few frames.

Any chance to reconstruct moving objects or it is just designed for static scenes?
